core/src/main/scala/org/apache/spark/resource/ResourceProfile.scala

```diff
@@ -402,7 +420,7 @@ object ResourceProfile extends Logging {
   }
 
   private[spark] case class ExecutorResourcesOrDefaults(
-      cores: Int,
+      cores: Option[Int], // Can only be None for standalone cluster.
```

Comment:
standalone and local-cluster?

Reply:
I think this is for multiple resource profiles/stage level scheduling. Currently we didn't add the support for local-cluster.

Reply:
Thanks, done.